web/src/connectors/task-annotator-connector/task-annotator-utils.ts

```diff
@@ -96,6 +96,13 @@ const addTableValues = (ann: Annotation): TableApi => {
     };
 };
 
+const createText = (token: PageInfoObjs | PageToken) => {
+    const previousSymbol = token?.previous || '';
+    const afterSymbol = token?.after || '';
+    const text = token?.text || '';
+    return previousSymbol + text + afterSymbol;
```

Comment:
I'm not sure we need previous symbols in this attr as it's some spaces for paragraphs etc. I think afterSymbols should be enough here

Reply:
It can be different symbols in the previous field. I`m afraid of the risks of throwing this functionality

Reply:
ok, let's leave it as it is